src/intTest/resources/outbound_uat_data/amendment/live-1.fhir.ignore.json

```diff
@@ -29,6 +29,11 @@
       "op": "replace",
       "path": "/address/0/line/4",
       "value": "KENT"
+    },
```

Comment:
live-1 only contains addresses. Can this also be enabled?

Reply:
live-1 contains also haCipher, that cannot be proceed right now. I get error that: "Couldn't decode recipient: KC01"